lib/web/forms/registration.go

```diff
@@ -0,0 +1,71 @@
+package webforms
+
+import (
+	"errors"
+
+	"go-google-scraper-challenge/constants"
+	"go-google-scraper-challenge/helpers"
+	"go-google-scraper-challenge/lib/models"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+)
+
+type RegistrationForm struct {
+	Email                string `form:"email"`
+	Password             string `form:"password"`
+	PasswordConfirmation string `form:"password_confirmation"`
+}
+
+func (f RegistrationForm) Validate() (valid bool, err error) {
+	err = validation.ValidateStruct(&f,
+		validation.Field(&f.Email, validation.Required, is.EmailFormat),
+		validation.Field(&f.Password, validation.Required),
+		validation.Field(&f.PasswordConfirmation, validation.Required.When(f.Password != ""), validation.By(f.validatePasswordConfirmation(f.Password))),
```

Comment:
Then we can omit this `validation.Required.When(f.Password != "")` out when we're checking with the password length